server/src/lib/model/db/migrations/20191015151914-add-teams-and-user-client-teams.ts

```diff
@@ -0,0 +1,28 @@
+export const up = async function(db: any): Promise<any> {
+  return db.runSql(
+    `
+    CREATE TABLE teams (
+      team_id CHAR(36) NOT NULL PRIMARY KEY,
+      teamname TEXT NOT NULL DEFAULT (''),
+      leader CHAR(36) NOT NULL,
+      invite_link TEXT DEFAULT NULL DEFAULT (''),
```

Comment:
I'd go for a name like `invite_token` since this probably wouldn't/shouldn't hold a whole URL

Reply:
what does the token looks like? will it be more than , like, 255 chars?
I am thinking maybe the token is unique and of type varchar(255).